GameServerLib/Chatbox/Commands/TargetableCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿using GameServerCore;
+
+namespace LeagueSandbox.GameServer.Chatbox.Commands
+{
+    public class TargettableCommand : ChatCommandBase
+    {
+        private readonly IPlayerManager _playerManager;
+        public override string Command => "targetable";
+        public override string Syntax => $"{Command} false (untargetable) / true (targetable)";
+
+        public TargettableCommand(ChatCommandManager chatCommandManager, Game game)
+            : base(chatCommandManager, game)
+        {
+            _playerManager = game.PlayerManager;
+        }
+
+        public override void Execute(int userId, bool hasReceivedArguments, string arguments = "")
+        {
+            var split = arguments.ToLower().Split(' ');
+
+            if (split.Length < 2 || !bool.TryParse(split[1], out var input))
```

Comment:
should be `split.Length != 2`
`input` is a bad variable name for a result var, maybe call it `parsed` or `userInput` something along these lines? maybe it can be set to `null` even? (not sure on this last one)